pkg/plugin/framework/server_mux.go

```diff
@@ -63,7 +76,7 @@ func (m *serverMux) getHandler(name string) (interface{}, error) {
 
 	initializer, found := m.initializers[name]
 	if !found {
-		return nil, errors.Errorf("unknown %v plugin: %s", m.kind, name)
+		return nil, errors.Errorf("%v plugin: %s not found or had invalid format. Valid format: <namespace>/<name>", m.kind, name)
```

Comment:
I don't think the first component is a namespace (`velero.io` isn't). I think it's mean to be a domain name, if you're following annotation/label key conventions.

Reply:
I'm trying to indicate that the second component might have an invalid format. Otherwise I am misinterpreting what you said, please say it in a different way?

Reply:
I cleaned up the message a bit.

Reply:
I read `<namespace>/<name>` as `namespace = velero.io` and `name = job`. In `velero.io/job`, I wouldn't necessarily called `velero.io` a namespace, mostly because that's a specific thing in Kubernetes.

Was the intent to make the first part a vendor identifier?

Reply:
So `namespace` is a concept I just added. And yes, the idea is to be a vendor identifier. Maybe the issue will give better context: https://github.com/heptio/velero/issues/583

Reply:
Yeah, the issues helps, thanks :)

There's a similar concept for annotations, labels, and finalizers. (like `velero.io/schedule-name`). As an example, for finalizers, they expect the names to be fully qualified domain names.

This is the error I get if I add one that's not an FQDN: ` metadata.finalizers[0]: Invalid value: "velero": name is neither a standard finalizer name nor is it fully qualified`. Unfortunately, [the docs](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers) don't specify that restriction :(

Reply:
FYI - here's what I found re: labels -- https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set

Reply:
I think then we should call `velero.io` a prefix and validate it to be a FQDN. We shouldn't call it a lable because `labels do not provide uniqueness` (as per that ^ doc). But we can say it's a key/value pair of prefix/name. Yeay/nay?

Reply:
Yeah, it's not actually a label, but it's a part of a string in a very similar vein. I like "prefix" or "vendor prefix," to be a little more specific. Vendors are then expected to use their own for their plugins.